### ISSUE TITLE
Research JavaFX Panes, Sizing, Alignment and Scalability

### ISSUE THREAD

**Issue by SasquatchMighty** [rsanchez-wsu/jfiles#106]
Soon we will begin placing features into our GUI. To avoid rewriting code we need to plan ahead. Specifically we need to select the appropriate JavaFX panes to ensure our features are easy to place and position. Then we need to polish our features by correcting Sizing, Alignment and Scalability. To ensure we select the best JavaFX components for our needs we should begin by researching the possible options. A recommended resource is http://docs.oracle.com/javafx/2/layout/jfxpub-layout.htm. Our findings and suggestions should be posted to the Team 6 Wiki, possibly under a new page. Keep in mind our current pictorial GUI representation.
(Preliminary Picture of the GUI Layout curtesy of erik from Slack https://ceg3120fall2016.slack.com/messages/gui_collaboration/ )
![erik_early_gui_mock_up](https://cloud.githubusercontent.com/assets/14119599/19458194/9811a1fe-9498-11e6-83b2-a9b5b43b80fa.jpg)

**Comment by SasquatchMighty** [comment]
We did not post anything on the Wiki but we have moved past this issue. In the commits eb3a8ed, 86d1005 and 32be927 We used a BorderPane as a foundation. We then stacked HBoxs, VBoxs and StackPanes on top of the BorderPane to ensure everything is in a proper layout. Infact we are three pains and four labels deep in one particular spot. I almost exclusivle drew my research from http://docs.oracle.com/javafx/2/layout/jfxpub-layout.htm.
